### PR TITLE
Added top level readme for copilot chat.

### PR DESCRIPTION
**Pull request by microsoftShannon**
### Motivation and Context
To create clarity for onboarding of Copilot Chat sample app.

### Description
Added readme and two images to Copilot Chat directory.

### CODE REVIEW

File: samples/apps/copilot-chat-app/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Copilot Chat Sample Application
+>! IMPORTANT  This learning sample is for educational purposes only and should not be used in any production use case.  It is intended to highlight concepts of Semantic Kernel and not any architectural / Security design practices to be used.
+
+## About the Copilot 
+The Copilot Chat sample allows you to build your own integrated large language model chatbot.  This is an enriched intelligence app, with multiple dynamic components including command messages, user intent, and memories.  The chat prompt and response will evolve as the conversation between the user and the application proceeds.  This chat experience is a chat skill containing multiple functions that work together to construct the final prompt for each exchange.
```
Comment (adrianwyatt) [inline samples/apps/copilot-chat-app/README.md:5]: For readability outside of markdown rendering, I recommend adding line endings after about 80-120 characters - on sentence boundaries when possible. The markdown will still render the same, but it makes it much easier to read in code.

File: samples/apps/copilot-chat-app/README.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+
+![UI Sample](images/UI-Sample.png)
+
+## Dependencies:
+
+Before following these instructions, please ensure your development environment and these components are functional:
```
Comment (adrianwyatt) [inline samples/apps/copilot-chat-app/README.md:12]: Let's add links to install pages on the keywords in these steps. For example:
```
4. [Node.js](https://nodejs.org/en/download) installed.
```

Comment (adrianwyatt) [inline samples/apps/copilot-chat-app/README.md:12]: [nit] I would like to remove assumptions in these statements. For example,
```
1.	[Visual Studio Code](https://code.visualstudio.com/Download)
2.	[Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
3.	[.NET 6.0](https://dotnet.microsoft.com/en-us/download/dotnet/6.0)
4.	[Node.js](https://nodejs.org/en/download)
5.	[Yarn](https://classic.yarnpkg.com/lang/en/docs/install)
```

File: samples/apps/copilot-chat-app/README.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+1.	For this sample application, it is assumed that you already have VS Code installed as your IDE.
+2.	As the repository is stored on GitHub, Git should be installed as well.  
+3.	The sample application requires .NET 6.0 or greater.  
+4.	Node.js should be installed, 
+5.	Yarn will be required.
+
+
+## Running the Sample
+1.	You will need an Open AI Key or Azure Open AI Service Key for this sample.
```
Comment (adrianwyatt) [inline samples/apps/copilot-chat-app/README.md:21]: Open AI keys: https://platform.openai.com/account/api-keys
AOI: ??

File: samples/apps/copilot-chat-app/README.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+2.	You will need to register your application in the Azure Portal.  Follow the steps to register your app here.
```
Comment (adrianwyatt) [inline samples/apps/copilot-chat-app/README.md:22]: "your application" is confusing in the context of our application that they are trying to use. Let's use the term "an application registration".

Comment (adrianwyatt) [inline samples/apps/copilot-chat-app/README.md:22]: LEt's add a link to the quickstart: https://learn.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app

File: samples/apps/copilot-chat-app/README.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+    1. Select Single-page application (SPA) as platform type, and the redirect URI will be `http://localhost:3000`
+    2.	Select `Accounts in any organizational directory and personal Microsoft Accounts` as supported account types for this sample.
+    3. Make a note of this Application (client) ID from the Azure Portal, we will make use of it later.
+3.	The sample uses two applications, a front-end web UI, and a back-end server.  First, let’s set up and verify the back-end server is running.
```
Comment (adrianwyatt) [inline samples/apps/copilot-chat-app/README.md:27]: Could we use "back-end API server" ?

File: samples/apps/copilot-chat-app/README.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+    1. Navigate to `\samples\apps\copilot-chat-app\SKWebApi`
+    2.	Update `appsettings.json` with these settings:
+
+          *	Under the `“ServicePort”` block, make the following configuration changes to match your instance:
```
Comment (adrianwyatt) [inline samples/apps/copilot-chat-app/README.md:32]: `ServicePort` should be `CompletionConfig`

File: samples/apps/copilot-chat-app/README.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+            * `“AIService”: “AzureOpenAI”`, or whichever option is appropriate for your instance.
+            * `“DeploymentOrModelID”: “text-davinci-003”,` or whichever option is appropriate for your instance.  
+            * `“Endpoint”:` “Your Endpoint address, i.e. http://project.region.azure.com”
+            * `“Key”:` “Your azure endpoint key, i.e. a 36+ digit hex code”
```
Comment (adrianwyatt) [inline samples/apps/copilot-chat-app/README.md:37]: We do not want anyone putting their key in this file. Recomment using "dotnet user-secrets" for running locally.


Open a command prompt and go to the `samples/apps/copilot-chat-app/SKWebApi` directory. Then run the following commands:
```
dotnet user-secrets set "CompletionConfig:Key" "YOUR KEY OPENAI or AZURE OPENAI KEY"
dotnet user-secrets set "EmbeddingConfig:Key" "YOUR KEY OPENAI or AZURE OPENAI KEY"
```

File: samples/apps/copilot-chat-app/README.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+
+            * `“AIService”: “AzureOpenAI”`, or whichever option is appropriate for your instance.
+            * `“DeploymentOrModelID”: “text-davinci-003”,` or whichever option is appropriate for your instance.  
+            * `“Endpoint”:` “Your Endpoint address, i.e. http://project.region.azure.com”
+            * `“Key”:` “Your azure endpoint key, i.e. a 36+ digit hex code” 
+
+        * Under the `“EmbeddingConfig”` block, make sure the following configuration changes to match your instance:
+            * `“AIService”: “AzureOpenAI”,` or whichever option is appropriate for your instance.
+            * `“DeploymentOrModelID”: “text-embedding-ada-002”,` or whichever option is appropriate for your instance.    
+            *	`“Key”:` “Your azure endpoint key, i.e. a 36+ digit hex code”
```
Comment (adrianwyatt) [inline samples/apps/copilot-chat-app/README.md:42]: Same comment as above.

File: samples/apps/copilot-chat-app/README.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+3.	The sample uses two applications, a front-end web UI, and a back-end server.  First, let’s set up and verify the back-end server is running.
+
+    1. Navigate to `\samples\apps\copilot-chat-app\SKWebApi`
+    2.	Update `appsettings.json` with these settings:
+
+          *	Under the `“ServicePort”` block, make the following configuration changes to match your instance:
+
+            * `“AIService”: “AzureOpenAI”`, or whichever option is appropriate for your instance.
+            * `“DeploymentOrModelID”: “text-davinci-003”,` or whichever option is appropriate for your instance.  
+            * `“Endpoint”:` “Your Endpoint address, i.e. http://project.region.azure.com”
```
Comment (adrianwyatt) [inline samples/apps/copilot-chat-app/README.md:36]: For example endpoints, let's use Contoso: `https://contoso.openai.azure.com`.

File: samples/apps/copilot-chat-app/README.md (hunk 7 above)
Comment (adrianwyatt) [reply samples/apps/copilot-chat-app/README.md:37]: These instuctions might make sense in the next section below ("Build the back-end server"). Maybe just make a reference here to setting the keys in that section.

File: samples/apps/copilot-chat-app/README.md [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+    4.	Test the back-end server to confirm it is running.
+        * Open a web browser, and navigate to `https://localhost:40443/probe`
+        * You should see a confirmation message: `Semantic Kernel service is up and running`
+>Note: you may need to accept the locally signed certificate on your machine in order to see this message.  It is important to do this, as your browser may need to accept the certificate before allowing the WebApp to communicate with the backend.
+
+>Note: You may need to acknowledge the Windows Defender Firewall, and allow the app to communicate over private or public netowrks as appropriate.
+
+
+5.	Now that the back-end server is setup, and confirmed operating, let’s proceed with setting up the front-end WebApp.
+    1. Navigate to `\apps\copilot-chat-app\webapp`
```
Comment (adrianwyatt) [inline samples/apps/copilot-chat-app/README.md:57]: There are spacing inconsistencies in this section.

File: samples/apps/copilot-chat-app/README.md [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+    2.	Copy `.env.example` into a new file with the name “`.env`” and make the following configuration changes to match your instance:
+    3. Use the Application (client) ID from the Azure Portal steps above and paste the GUID into the .env file next to `REACT_APP_GRAPH_CLIENT_ID= `
+    4.	Execute the command `Yarn Install`
+    5.	Execute the command `Yarn Start`
+
```
Comment (adrianwyatt) [inline samples/apps/copilot-chat-app/README.md:62]: extra line

File: samples/apps/copilot-chat-app/README.md [hunk 12]
```diff
@@ -0,0 +1,77 @@
...
+5.	Now that the back-end server is setup, and confirmed operating, let’s proceed with setting up the front-end WebApp.
+    1. Navigate to `\apps\copilot-chat-app\webapp`
+    2.	Copy `.env.example` into a new file with the name “`.env`” and make the following configuration changes to match your instance:
+    3. Use the Application (client) ID from the Azure Portal steps above and paste the GUID into the .env file next to `REACT_APP_GRAPH_CLIENT_ID= `
+    4.	Execute the command `Yarn Install`
+    5.	Execute the command `Yarn Start`
+    
+    6. Wait for the startup to complete.
+    7. With the back end and front end running, your web browser should automatically launch and navigate to `https://localhost:3000`
+    8. Sign in in with your account details.
```
Comment (adrianwyatt) [inline samples/apps/copilot-chat-app/README.md:65]: Should we mention this needs to be a Microsoft work or personal account?

File: samples/apps/copilot-chat-app/README.md [hunk 13]
```diff
@@ -0,0 +1,77 @@
...
+    9. Grant permission to use your account details.
```
Comment (adrianwyatt) [inline samples/apps/copilot-chat-app/README.md:66]: Let's be crisp about privacy here: "Grant permission for the application, this is normally just to read your account name".

File: samples/apps/copilot-chat-app/README.md [hunk 14]
```diff
@@ -0,0 +1,77 @@
...
+    10.	If it fails to load, you may need to validate the certificate of the backend server.  If you see an error message, or if the machine appears to stop, waiting for the connection to the backend server, please see the troubleshooting section to accept your machines certificates.
```
Comment (adrianwyatt) [inline samples/apps/copilot-chat-app/README.md:67]: Let's pair this down to something like "If you experience any errors or issues, consult the troubleshooting section below."

**Review by adrianwyatt — CHANGES_REQUESTED** [review]

File: samples/apps/copilot-chat-app/README.md (hunk 1 above)
Comment (microsoftShannon) [reply samples/apps/copilot-chat-app/README.md:5]: Done.

File: samples/apps/copilot-chat-app/README.md (hunk 2 above)
Comment (microsoftShannon) [reply samples/apps/copilot-chat-app/README.md:12]: Done.

File: samples/apps/copilot-chat-app/README.md (hunk 4 above)
Comment (microsoftShannon) [reply samples/apps/copilot-chat-app/README.md:22]: Updated

File: samples/apps/copilot-chat-app/README.md (hunk 12 above)
Comment (microsoftShannon) [reply samples/apps/copilot-chat-app/README.md:65]: updated.

File: samples/apps/copilot-chat-app/README.md (hunk 14 above)
Comment (microsoftShannon) [reply samples/apps/copilot-chat-app/README.md:67]: done

File: samples/apps/copilot-chat-app/README.md (hunk 13 above)
Comment (microsoftShannon) [reply samples/apps/copilot-chat-app/README.md:66]: updated

File: samples/apps/copilot-chat-app/README.md (hunk 2 above)
Comment (microsoftShannon) [reply samples/apps/copilot-chat-app/README.md:12]: Updated and removed assumptions as well.

File: samples/apps/copilot-chat-app/README.md (hunk 6 above)
Comment (microsoftShannon) [reply samples/apps/copilot-chat-app/README.md:32]: Did that change today?  Updated.

File: samples/apps/copilot-chat-app/README.md (hunk 9 above)
Comment (microsoftShannon) [reply samples/apps/copilot-chat-app/README.md:36]: Updated

File: samples/apps/copilot-chat-app/README.md (hunk 4 above)
Comment (microsoftShannon) [reply samples/apps/copilot-chat-app/README.md:22]: Good catch.

File: samples/apps/copilot-chat-app/README.md (hunk 7 above)
Comment (microsoftShannon) [reply samples/apps/copilot-chat-app/README.md:37]: Updated

File: samples/apps/copilot-chat-app/README.md (hunk 8 above)
Comment (microsoftShannon) [reply samples/apps/copilot-chat-app/README.md:42]: updated.

File: samples/apps/copilot-chat-app/README.md (hunk 10 above)
Comment (microsoftShannon) [reply samples/apps/copilot-chat-app/README.md:57]: Resolved.

File: samples/apps/copilot-chat-app/README.md (hunk 11 above)
Comment (microsoftShannon) [reply samples/apps/copilot-chat-app/README.md:62]: Resolved

File: samples/apps/copilot-chat-app/README.md [hunk 15]
```diff
@@ -0,0 +1,78 @@
...
+    6.	Test the back-end server to confirm it is running.
+        * Open a web browser, and navigate to `https://localhost:40443/probe`
+        * You should see a confirmation message: `Semantic Kernel service is up and running`
+>Note: you may need to accept the locally signed certificate on your machine in order to see this message.  It is important to do this, as your browser may need to accept the certificate before allowing the WebApp to communicate with the backend.
+>Note: You may need to acknowledge the Windows Defender Firewall, and allow the app to communicate over private or public netowrks as appropriate.
+5.	Now that the back-end API server is setup, and confirmed operating, let’s proceed with setting up the front-end WebApp.
+    1. Navigate to `\apps\copilot-chat-app\webapp`
+    2.	Copy `.env.example` into a new file with the name “`.env`” and make the following configuration changes to match your instance:
+    3. Use the Application (client) ID from the Azure Portal steps above and paste the GUID into the .env file next to `REACT_APP_GRAPH_CLIENT_ID= `
+    4.	Execute the command `Yarn Install`
```
Comment (glahaye) [inline samples/apps/copilot-chat-app/README.md:61]: Nit:  these two commands should be all in lower case.  On Windows, it doesn't matter much, but it does on Mac and Linux

Comment (adrianwyatt) [reply samples/apps/copilot-chat-app/README.md:61]: nice catch!

**Review by adrianwyatt — APPROVED** [review]

**Pull request merged**